src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -90,8 +90,7 @@ const DragDropcontext: FC<dragDropProps> = ({
           method: 'patch',
           data,
         });
-
-        toast.success(`${updateTaskResult.data.message}`, {
+        toast.success(updateTaskResult.data.message ? updateTaskResult.data.message : 'Sucessfully Assigned Task', {
```

Comment:
We don't have any message from the response when its PATCH request is successful

Reply:
Let's remove the `updateTaskResult.data.message` and just keep hard coded message.
Just checked the API contracts `204` doesn't return a response 👍🏽 
https://github.com/Real-Dev-Squad/website-api-contracts/tree/main/tasks#patch-tasksid